### ISSUE TITLE
Change tab focus in autopilot

### ISSUE THREAD

**Issue by Elblinator** [OpenSlides/openslides-client#6139]
**Describe the behaviour**
If your focus in on a running poll and the poll is changed to finished/publiched your focus jumps to the projector (the element below)

**How to Reproduce**
1. be in the autopilot with a running poll 
2. have your focus on the running poll
3. with a different device change the poll to finished/published
4. yout focus now is on the projector

**Wanted behavior**
your focus should be on the finished poll and not an elemnet below
